### PR TITLE
[prod] March 2024 - 28.0.4, 27.1.8 & 26.0.13

### PR DESCRIPTION
**Pull request by skjnldsv**
https://github.com/nextcloud-releases/updater_server/pull/1021

### CODE REVIEW

File: config/config.php [hunk 1]
```diff
@@ -110,98 +110,98 @@
...
 		'26' => [
 			'100' => [
-				'latest' => '26.0.12',
-				'internalVersion' => '26.0.12.2',
-				'downloadUrl' => 'https://download.nextcloud.com/server/releases/nextcloud-26.0.12.zip',
+				'latest' => '26.0.13',
+				'internalVersion' => '26.0.13.1',
+				'downloadUrl' => 'https://download.nextcloud.com/server/releases/nextcloud-26.0.13.zip',
 				'web' => 'https://docs.nextcloud.com/server/26/admin_manual/maintenance/upgrade.html',
 				'eol' => false,
```
Comment (nickvergessen) [inline config/config.php:181]:
```suggestion
				'eol' => true,
```

File: config/config.php [hunk 2]
```diff
@@ -110,98 +110,98 @@
...
 		'25.0.13' => [
 			'100' => [
-				'latest' => '26.0.12',
-				'internalVersion' => '26.0.12.2',
-				'downloadUrl' => 'https://download.nextcloud.com/server/releases/nextcloud-26.0.12.zip',
+				'latest' => '26.0.13',
+				'internalVersion' => '26.0.13.1',
+				'downloadUrl' => 'https://download.nextcloud.com/server/releases/nextcloud-26.0.13.zip',
 				'web' => 'https://docs.nextcloud.com/server/26/admin_manual/maintenance/upgrade.html',
 				'eol' => false,
```
Comment (nickvergessen) [inline config/config.php:197]:
```suggestion
				'eol' => true,
```

File: config/config.php [hunk 3]
```diff
@@ -731,98 +731,98 @@
...
 		'26' => [
 			'100' => [
-				'latest' => '26.0.13 RC1',
-				'internalVersion' => '26.0.13.0',
-				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-26.0.13rc1.zip',
+				'latest' => '26.0.13',
+				'internalVersion' => '26.0.13.1',
+				'downloadUrl' => 'https://download.nextcloud.com/server/releases/nextcloud-26.0.13.zip',
 				'web' => 'https://docs.nextcloud.com/server/26/admin_manual/maintenance/upgrade.html',
 				'eol' => false,
```
Comment (nickvergessen) [inline config/config.php:802]:
```suggestion
				'eol' => true,
```

File: config/config.php [hunk 4]
```diff
@@ -731,98 +731,98 @@
...
 		'25.0.13' => [
 			'100' => [
-				'latest' => '26.0.13 RC1',
-				'internalVersion' => '26.0.13.0',
-				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-26.0.13rc1.zip',
+				'latest' => '26.0.13',
+				'internalVersion' => '26.0.13.1',
+				'downloadUrl' => 'https://download.nextcloud.com/server/releases/nextcloud-26.0.13.zip',
 				'web' => 'https://docs.nextcloud.com/server/26/admin_manual/maintenance/upgrade.html',
 				'eol' => false,
```
Comment (nickvergessen) [inline config/config.php:818]:
```suggestion
				'eol' => true,
```

File: tests/integration/features/beta.feature [hunk 5]
```diff
@@ -364,18 +364,18 @@ Feature: Testing the update scenario of beta releases
     And the installation mtime is "11"
     When The request is sent
     Then The response is non-empty
-    And Update to version "26.0.13.0" is available
-    And URL to download is "https://download.nextcloud.com/server/prereleases/nextcloud-26.0.13rc1.zip"
+    And Update to version "26.0.13.1" is available
+    And URL to download is "https://download.nextcloud.com/server/releases/nextcloud-26.0.13.zip"
     And URL to documentation is "https://docs.nextcloud.com/server/26/admin_manual/maintenance/upgrade.html"
     And EOL is set to "0"
```
Comment (nickvergessen) [inline tests/integration/features/beta.feature:370]:
```suggestion
    And EOL is set to "1"
```

File: tests/integration/features/beta.feature [hunk 6]
```diff
@@ -385,62 +385,62 @@ Feature: Testing the update scenario of beta releases
...
```
Comment (nickvergessen) [inline tests/integration/features/beta.feature:391]:
```suggestion
    And EOL is set to "1"
```

File: tests/integration/features/stable.feature [hunk 7]
```diff
@@ -449,18 +449,18 @@ Feature: Testing the update scenario of stable releases
     And the installation mtime is "11"
     When The request is sent
     Then The response is non-empty
-    And Update to version "26.0.12.2" is available
-    And URL to download is "https://download.nextcloud.com/server/releases/nextcloud-26.0.12.zip"
+    And Update to version "26.0.13.1" is available
+    And URL to download is "https://download.nextcloud.com/server/releases/nextcloud-26.0.13.zip"
     And URL to documentation is "https://docs.nextcloud.com/server/26/admin_manual/maintenance/upgrade.html"
     And EOL is set to "0"
```
Comment (nickvergessen) [inline tests/integration/features/stable.feature:455]:
```suggestion
    And EOL is set to "1"
```

File: tests/integration/features/stable.feature [hunk 8]
```diff
@@ -470,39 +470,39 @@ Feature: Testing the update scenario of stable releases
     And the installation mtime is "71"
     When The request is sent
     Then The response is non-empty
-    And Update to version "26.0.12.2" is available
-    And URL to download is "https://download.nextcloud.com/server/releases/nextcloud-26.0.12.zip"
+    And Update to version "26.0.13.1" is available
+    And URL to download is "https://download.nextcloud.com/server/releases/nextcloud-26.0.13.zip"
     And URL to documentation is "https://docs.nextcloud.com/server/26/admin_manual/maintenance/upgrade.html"
     And EOL is set to "0"
```
Comment (nickvergessen) [inline tests/integration/features/stable.feature:476]:
```suggestion
    And EOL is set to "1"
```

**Review by AndyScherzinger — APPROVED** [review]

**Comment by skjnldsv** [comment]
@nickvergessen let's do it in a separate PR

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**